packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -1049,14 +1049,69 @@ describe('TransactionController', () => {
 
       expect(controller.state.transactions).toHaveLength(0);
     });
+
+    it('removes only txs with given address on current network', async () => {
```

Comment:
Is it best we have two tests to cover both permutations?

One for only an address filter, and one for an address and network filter?

Reply:
I've split the both test cases